src/build_targets.rs

```diff
@@ -81,7 +81,8 @@ impl BuildTargets {
             | ("freebsd", _)
             | ("dragonfly", _)
             | ("netbsd", _)
-            | ("android", _) => {
+            | ("android", _)
+            | ("haiku", _)=> {
```

Comment:
```suggestion
            | ("haiku", _) => {
```

Otherwise the fmt check would fail.

Reply:
Yeah saw this one earlier, not right, I've run `rust fmt` in the source directory just earlier, will do another round in the morning and see if I can push the changes made by rust for this.